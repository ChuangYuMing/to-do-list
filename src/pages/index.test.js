import React from "react";
import { render } from "@testing-library/react";
import Index from "./Index";

it("renders correctly", () => {
  const { asFragment } = render(<Index />);
  expect(asFragment()).toMatchSnapshot();
});
