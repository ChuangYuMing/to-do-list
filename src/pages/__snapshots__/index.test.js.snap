// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <main
    class="main className"
  >
    <h1
      class="title"
    >
      To-Do List
    </h1>
    <div
      class="add-task-wrap"
    >
      <input
        class="add-task-input"
        placeholder="Add a new task"
        type="text"
        value=""
      />
      <button
        class="add-task-button"
      >
        Add
      </button>
    </div>
    <div
      class="search-task-wrap"
    >
      <input
        class="search-task-input"
        placeholder="Search tasks"
        type="text"
        value=""
      />
      <div
        class="checkbox-wrap"
      >
        <input
          class="checkbox-input"
          id="sort-by-created-date"
          name="sort-by-created-date"
          type="checkbox"
        />
        <label
          class="checkbox-label"
          for="sort-by-created-date"
        >
          Sort by created date
        </label>
      </div>
    </div>
    <div
      class="filter-task-wrap"
    >
      <span
        class="active"
        data-status="All"
      >
        All
      </span>
      <span
        class=""
        data-status="Uncompleted"
      >
        Uncompleted
      </span>
      <span
        class=""
        data-status="Completed"
      >
        Completed
      </span>
    </div>
    <div
      class="task-list-wrap"
    />
  </main>
</DocumentFragment>
`;
